provider/kubernetes/kubernetes.go

```diff
@@ -315,6 +309,48 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 	return &templateObjects, nil
 }
 
+func getEntrypoints(i *v1beta1.Ingress) []string {
+	entrypointsAnnotation, ok := i.Annotations[types.LabelFrontendEntryPoints]
+	if ok {
+		return strings.Split(entrypointsAnnotation, ",")
+	}
+	return nil
```

Comment:
If the `nil` slice is handled equally to the empty slice case by Traefik, then we don't need the extra `ok` check it seems?

Reply:
`strings.Split` return a non empty slice when his first arg is empty, so if i didn't check extra `ok` , i will need to check if first of slice is empty. It seem more maintainable that way, no?

Reply:
Good point, I missed that property of `strings.Split`. Let's keep things like they are right now then.